.github/workflows/homebrew-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Homebrew Tap
+on:
+  release:
+    types: [released]
```

Comment:
Note that this will only trigger on _releases_ and NOT _pre_releases.

Reply:
This works for me. We can make our release process trigger a pre-release, do some initial testing, then push out the full release after we are happy with the pre-release.